packages/blade/src/components/ProgressBar/_decisions/decisions.md

```diff
@@ -53,9 +55,11 @@ import { ProgressBar } from '@razorpay/components';
 - We will set `aria-role` as `progressbar` for `progress` variant (`accessibilityRole` for React Native)
 - We will set `aria-role` as `meter` for `meter` variant (`accessibilityRole` for React Native)
 - We will pass the value of `value` prop as `aria-valuenow` (`accessibilityValue` for React Native)
-- Pass `aria-busy` as `true` (`accessibilityState` for React Native)
-- We will set `aria-valuemin` as 0
-- We will set `aria-valuemax` as 100
+- We will set `aria-valuemin` as `min` (default as `0`)
+- We will set `aria-valuemax` as `max` (default as `100`)
+- We will set `aria-valuenow` as clamped `value` (If value passed is less than min, min would be used as the current value. If value passed is greater than max, max would be used as the current value)
+- We will set `aria-valuetext` as clamped percentage `value` when variant is `progress` (`20` will be passed as `20%`)
```

Comment:
Don't know how screen readers read the `%` symbol so maybe we can pass `20 percent`